### ISSUE TITLE
Support token replacements in html templates for custom content definitions when building policies

### ISSUE THREAD

**Issue by rhendrie** [azure-ad-b2c/vscode-extension#62]
Would be useful to support environment substitution in HTML templates for custom content definitions.  If, for example, I was to create a sub-folder called templates in the same folder as the policy xml files and run the Policy Build tool for an environment it could apply the environment transformations to those files also to replace things like absolute references to resources in the storage account hosting to the content (e.g. images, styles etc. )

**Comment by trittimo** [comment]
If you want, you can adapt the included powershell script [here on line 32](https://github.com/azure-ad-b2c/vscode-extension/blob/8c78e56525f5f7367052721c59b3d577edff6beb/scripts/Build-CustomPolicies.ps1#L32) to include all files instead of just XML and use that as a build tool. Given that the plugin can't currently upload any assets other than B2C policies, using that powershell script would probably be more desirable as you can automate that in a build pipeline or similar.

**Comment by rendermouse** [comment]
I agree it would be nice to support html templates in the project, and update settings vars within them, such as link URLs to dev/prod ForgotPassword flow from the Login page templates, etc.
